lib/node-utils.ts

```diff
@@ -220,3 +224,29 @@ export function isRenderVariableDeclarator(
 
   return false;
 }
+
+// TODO: extract into types file?
+export type ModuleImportation =
+  | TSESTree.ImportDeclaration
+  | TSESTree.CallExpression;
+
+export function getImportModuleName(
+  node: ModuleImportation | undefined | null
+): string | undefined {
+  // import node of shape: import { foo } from 'bar'
+  if (
+    node?.type === AST_NODE_TYPES.ImportDeclaration &&
```

Comment:
we've usually centered this checks on `isCallExpression` in `node-utils`

Reply:
ups sorry, this comment was supposed to be in line 246. For this check there's no function, but you could add it (there are all type guards)

Reply:
Actually I was thinking about adding a generic one, as I'm tired of having one function per check 😂 

Shouldn't be possible to implement some method like this?

```typescript
function getIsNodeType(
  node: TSESTree.Node,
  expectedType: keyof typeof AST_NODE_TYPES
): boolean {
  return node?.type === AST_NODE_TYPES[expectedType];
}
```

ccing our TS expert @timdeschryver 😅

Reply:
the problem with that is that you lose the type inference I think, the `x is TheNodeTypeYouConfirm`  from the [type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html)

Reply:
I guess I could extract `keyof tyepof AST_NODE_TYPES` into `T` and update return type to `node is T` but to be honest no clue if I'm trying to go too far here. Probably won't work so I'll extract it into its own method in a new PR where I'll add tests for utils.

Reply:
Didn't realized we already had `isCallExpression`. Using this one now, and `isImportDeclaration` added too.